src/test/java/model/MoneyTest.java

```diff
@@ -0,0 +1,34 @@
+package model;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+class MoneyTest {
+
+    @Test
+    @DisplayName("1000원 이상이며 1000으로 나누어 떨어지는 금액 전달받을 때 오류 없이 돈을 생성할 수 있다.")
+    void canCreateMoney() {
+        // given
+        final String rightInput = "1000";
+
+        // when
+        final Money money = new Money(rightInput);
+
+        // then
+        assertThat(money.getMoney()).isEqualTo(Integer.parseInt(rightInput));
```

Comment:
@DisplayName 을 보면, 해당 테스트는 오류(예외) 없이 생성이 되는 것을 확인 하는 것이 주목적입니다.
따라서 
```suggestion
        // when,then
        assertThatCode(() -> new Money(rightInput)).doesNotThrowAnyException();
```

형태로 테스트 하는게 맞는 것 같습니다.

Reply:
https://pjh3749.tistory.com/241

위 링크를 참고하여 이런 기능도 있구나!를 깨닫고 노션에 간략하게 정리하였습니다!

Reply:
LottoNumberTest, MoneyTest, LottoTest에 해당 리뷰를 적용시켰습니다!